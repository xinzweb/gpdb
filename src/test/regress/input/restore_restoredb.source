-- start_ignore
-- find version info
select version();
\! pg_restore --version

\echo Test pg_restore
drop database if exists regressrestoretest9;
create database regressrestoretest9;
\! pg_restore -d regressrestoretest9 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc

drop database if exists regressrestoretest10;
create database regressrestoretest10;
\c regressrestoretest10
\i @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fp
\c regression

drop database if exists regressrestoretest11;
create database regressrestoretest11;
\! pg_restore -d regressrestoretest11 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_ft

drop database if exists regressrestoretest12;
create database regressrestoretest12;
\! pg_restore -d regressrestoretest12 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_Z

drop database if exists regressrestoretest13;
create database regressrestoretest13;
\! pg_restore -d regressrestoretest13 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_schema
\! pg_restore -d regressrestoretest13 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_data

drop database if exists regressrestoretest14;
create database regressrestoretest14;
\! pg_restore -d regressrestoretest14 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_blobs

drop database if exists regressrestoretest15;
create database regressrestoretest15;
\! pg_restore -d regressrestoretest15 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_create

drop database if exists regressrestoretest16;
create database regressrestoretest16;
\! pg_restore -d regressrestoretest16 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_enc

drop database if exists regressrestoretest17;
create database regressrestoretest17;
\! pg_restore -d regressrestoretest17 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_s6

drop database if exists regressrestoretest18;
create database regressrestoretest18;
\! pg_restore -d regressrestoretest18 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_nos6

drop database if exists regressrestoretest19;
create database regressrestoretest19;
\! pg_restore -d regressrestoretest19 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_t1

drop database if exists regressrestoretest20;
create database regressrestoretest20;
\! pg_restore -d regressrestoretest20 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_not1

drop database if exists regressrestoretest21;
create database regressrestoretest21;
\! pg_restore -d regressrestoretest21 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_oid

drop database if exists regressrestoretest22;
create database regressrestoretest22;
\! pg_restore -d regressrestoretest22 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_noowner

drop database if exists regressrestoretest23;
create database regressrestoretest23;
\! pg_restore -d regressrestoretest23 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_nopriv

drop database if exists regressrestoretest24;
create database regressrestoretest24;
\! pg_restore -d regressrestoretest24 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_sqlquote

drop database if exists regressrestoretest25;
create database regressrestoretest25;
\! pg_restore -d regressrestoretest25 @abs_builddir@/results/pgdump_datatype/regressbkuptest1_fc_gpsyntax

-- end_ignore

