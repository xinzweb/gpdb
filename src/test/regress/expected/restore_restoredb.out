-- start_ignore
-- find version info
select version();
                                                                                                                    version                                                                                                                    
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 PostgreSQL 8.2.4 (Greenplum Database cvs Head build dev build ) on i386-apple-darwin8.9.1, compiled by GCC i686-apple-darwin8-gcc-4.0.1 (GCC) 4.0.1 (Apple Computer, Inc. build 5367) compiled on Jun  5 2007 12:29:58 (with assert checking)
(1 row)

\echo Test pg_restore
Now test pg_restore
drop database if exists regressrestoretest9;
NOTICE:  database "regressrestoretest9" does not exist, skipping
create database regressrestoretest9;
\! pg_restore -d regressrestoretest9 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest10;
NOTICE:  database "regressrestoretest10" does not exist, skipping
create database regressrestoretest10;
\c regressrestoretest10
\i /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fp
/Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fp: No such file or directory
\c regression
drop database if exists regressrestoretest11;
NOTICE:  database "regressrestoretest11" does not exist, skipping
create database regressrestoretest11;
\! pg_restore -d regressrestoretest11 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_ft
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest12;
NOTICE:  database "regressrestoretest12" does not exist, skipping
create database regressrestoretest12;
\! pg_restore -d regressrestoretest12 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_Z
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest13;
NOTICE:  database "regressrestoretest13" does not exist, skipping
create database regressrestoretest13;
\! pg_restore -d regressrestoretest13 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_schema
pg_restore: [archiver] could not open input file: No such file or directory
\! pg_restore -d regressrestoretest13 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_data
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest14;
NOTICE:  database "regressrestoretest14" does not exist, skipping
create database regressrestoretest14;
\! pg_restore -d regressrestoretest14 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_blobs
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest15;
NOTICE:  database "regressrestoretest15" does not exist, skipping
create database regressrestoretest15;
\! pg_restore -d regressrestoretest15 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_create
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest16;
NOTICE:  database "regressrestoretest16" does not exist, skipping
create database regressrestoretest16;
\! pg_restore -d regressrestoretest16 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_enc
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest17;
NOTICE:  database "regressrestoretest17" does not exist, skipping
create database regressrestoretest17;
\! pg_restore -d regressrestoretest17 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_s6
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest18;
NOTICE:  database "regressrestoretest18" does not exist, skipping
create database regressrestoretest18;
\! pg_restore -d regressrestoretest18 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_nos6
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest19;
NOTICE:  database "regressrestoretest19" does not exist, skipping
create database regressrestoretest19;
\! pg_restore -d regressrestoretest19 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_t1
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest20;
NOTICE:  database "regressrestoretest20" does not exist, skipping
create database regressrestoretest20;
\! pg_restore -d regressrestoretest20 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_not1
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest21;
NOTICE:  database "regressrestoretest21" does not exist, skipping
create database regressrestoretest21;
\! pg_restore -d regressrestoretest21 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_oid
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest22;
NOTICE:  database "regressrestoretest22" does not exist, skipping
create database regressrestoretest22;
\! pg_restore -d regressrestoretest22 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_noowner
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest23;
NOTICE:  database "regressrestoretest23" does not exist, skipping
create database regressrestoretest23;
\! pg_restore -d regressrestoretest23 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_nopriv
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest24;
NOTICE:  database "regressrestoretest24" does not exist, skipping
create database regressrestoretest24;
\! pg_restore -d regressrestoretest24 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_sqlquote
pg_restore: [archiver] could not open input file: No such file or directory
drop database if exists regressrestoretest25;
NOTICE:  database "regressrestoretest25" does not exist, skipping
create database regressrestoretest25;
\! pg_restore -d regressrestoretest25 /Users/ftian/work/crashburn/cdb2/cdb-pg/src/test/regress/results/pgdump_datatype/bkuptest1_fc_gpsyntax
pg_restore: [archiver] could not open input file: No such file or directory
-- end_ignore
